scipy/stats/_qmc.py

```diff
@@ -336,10 +336,57 @@ def discrepancy(
                          f" {set(methods)!r}")
 
 
+def pmindist(
```

Comment:
Almost look like this should go into spatial. What do you think Tyler?

Reply:
Sure, that's fair, and maybe we could prefix the name with `_` so it is "informally private?" In the more distant future we could write detailed tests on the `spatial` side and then try to support more metrics for efficient "closest distance in a sample" if we need to. I don't know if it would ever become public on `spatial` side, but seems like a useful internal utility at least.

One other comment I have is that it is helpful to leave the review comment "resolution" to the maintainers so we can double check that items are resolved on our end rather than collapsing them right away, but minor point.

Reply:
OK, moved to `spatial` with a `_` prefix.